src/rp2_common/hardware_clocks/include/hardware/clocks.h

```diff
@@ -165,6 +165,15 @@ extern "C" {
  */
 void clocks_init(void);
 
+/*! \brief Re-initialise the clock hardware
+ *  \ingroup hardware_clocks
+ *
+ *  During a 'lightsleep()' some clocks may have been deliberately left running.
```

Comment:
Looks like `lightsleep()` is only a MicroPython function, not a Pico C-SDK function, so perhaps `'lightsleep()'` here should be changed to `"light sleep"` ?